### PR TITLE
chore(#87): update popmotion example

### LINKED ISSUE #87: Update Popmotion demo to use the latest version of Popmotion
We need to update the Popmotion demo to use the latest version of Popmotion. Steps include:

- Download the latest version of the library, and place it in the `examples/[library-name]` folder. Minified files preferred, but non-minified works.
    - The filename should include the version number in it (for example: `jquery-3.0.0.min.js`).
- Update the demo to use the updated library, including any necessary code changes to the demo.
- Verify that the demo works across a sample of modern browsers (like Firefox, Chrome, Safari, and Edge).
    - [Follow the instructions here](https://github.com/sparkbox/bouncy-ball/blob/master/contributing.md#development-workflow) to run a test version of the site locally.
- Commit your changes and submit a pull request.

### PR DESCRIPTION
**Pull request by Tomekmularczyk**
Updates popmotion to v8.7.5

Fixes #87

### CODE REVIEW

File: examples/popmotion/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 </head>
 <body>
   <div id="ball"></div>
-  <script src="https://unpkg.com/popmotion@8.1.0/dist/popmotion.global.min.js"></script>
+  <script src="https://unpkg.com/popmotion@8.7.5/dist/popmotion.global.min.js"></script>
```
Comment (bryanbraun) [inline examples/popmotion/index.html:19]: Hey @Tomekmularczyk, we'd actually like to move away from using CDN urls for these resources, by saving a copy of the file and committing it to the repo. I should have been a bit more clear about this in [the first bullet point in the issue description](https://github.com/sparkbox/bouncy-ball/issues/87#issue-708279707).

Could you also make that update?

File: examples/popmotion/script.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+const ballStyler = styler(document.getElementById('ball'));
 
 tween({
   to: 160,
   duration: 575,
   ease: easing.easeIn,
-  yoyo: Infinity
-}).start(ballStyler.set('y'));
+  yoyo: Infinity,
+}).start((v) => ballStyler.set('y', v));
```
Comment (bryanbraun) [inline examples/popmotion/script.js:9]: Nice updates. 👍

File: examples/popmotion/index.html (hunk 1 above)
Comment (Tomekmularczyk) [reply examples/popmotion/index.html:19]: ah, missunderstanding then, I'll change it

**Review by bryanbraun — APPROVED** [review]
Works great!

**Pull request merged**